### ISSUE TITLE
AOT and Rollup

### ISSUE THREAD

**Issue by harunurhan** [inspirehep/record-editor#155]
We should investigate AOT (partly there already) and Rollup to apply in the project correctly.

For instance for AOT `platformBrowser().bootstrapModuleFactory(AppModuleNgFactory);` should be the way to bootstrap app in `main.ts`.

This link is probably enough: https://angular.io/guide/aot-compiler

**Comment by jmartinm** [comment]
Yes, that would be a very good improvement. Just checked that guide and it looks very complete, we should have the JIT for development and AOT for production separation. Also for `angular-cli` maybe there are some specific things?
